samples/cpp/oneccl/sources/service.h

```diff
@@ -0,0 +1,804 @@
+/* file: service.h */
```

Comment:
Two copies of the same file complicates file modification & bug fixing. We can use one file for both or remove unnecessary content (apriori, etc.)